packages/fluxible/lib/FluxibleContext.js

```diff
@@ -113,6 +113,21 @@ function executeActionProxy(context, actionContext, action, payload, done) {
                 '` will only start after `' + currentActionDisplayName + '` is complete.');
         }
     }
+    context._plugins.forEach(function pluginsEach(plugin) {
```

Comment:
Can we accumulate the `plugExecuteAction` plugins upon context creation so that we don't have to iterate over all of the plugins for every `executeAction`. In most cases, this will be empty even though there will be multiple plugins installed (fetchr, perf, i13n, etc.) and I don't want to add to the overhead of each action since it is a hot path.